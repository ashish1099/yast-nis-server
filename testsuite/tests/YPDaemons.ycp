/*
 * YaST2: Modules testsuite
 *
 * Description:
 *   Testsuite for the YPDaemons function
 *
 * Authors:
 *   Dan Vesely <dan@suse.cz>
 *
 * $Id$
 */

{
    include "testsuite.ycp";
    include "nis_server/io.ycp";
    include "nis_server/routines.ycp";

    map READ = $[
        ];

    map WRITE = $[
        ];
    
    map EXECUTE = $[
        "target" : $[
            "bash_output" : $[
                "stdout" : "No Process",
                "stderr" : ""
                ]
            ]
        ];

    map user_settings = $[];

    TEST (``(YPDaemons (`none)), [$[], $[], EXECUTE], nil);
    TEST (``(YPDaemons (`slave)), [$[], $[], EXECUTE], nil);
    TEST (``(YPDaemons (`master)), [$[], $[], EXECUTE], nil);
    TEST (``(YPDaemons (`client)), [$[], $[], EXECUTE], nil);
}
