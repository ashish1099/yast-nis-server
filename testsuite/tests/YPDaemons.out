Execute	.target.bash_output "/etc/init.d/ypxfrd status" $["TERM":"raw"] $["stderr":"", "stdout":"No Process"]
Execute	.target.bash_output "/etc/init.d/yppasswdd status" $["TERM":"raw"] $["stderr":"", "stdout":"No Process"]
Execute	.target.bash_output "/etc/init.d/ypserv status" $["TERM":"raw"] $["stderr":"", "stdout":"No Process"]
Return	nil
Execute	.target.bash_output "/etc/init.d/ypxfrd status" $["TERM":"raw"] $["stderr":"", "stdout":"No Process"]
Execute	.target.bash_output "/etc/init.d/yppasswdd status" $["TERM":"raw"] $["stderr":"", "stdout":"No Process"]
Execute	.target.bash_output "/etc/init.d/ypserv status" $["TERM":"raw"] $["stderr":"", "stdout":"No Process"]
Execute	.target.bash_output "/etc/init.d/ypserv status" $["TERM":"raw"] $["stderr":"", "stdout":"No Process"]
Return	nil
Execute	.target.bash_output "/etc/init.d/ypxfrd status" $["TERM":"raw"] $["stderr":"", "stdout":"No Process"]
Execute	.target.bash_output "/etc/init.d/yppasswdd status" $["TERM":"raw"] $["stderr":"", "stdout":"No Process"]
Execute	.target.bash_output "/etc/init.d/ypserv status" $["TERM":"raw"] $["stderr":"", "stdout":"No Process"]
Execute	.target.bash_output "/etc/init.d/ypserv status" $["TERM":"raw"] $["stderr":"", "stdout":"No Process"]
Return	nil
Execute	.target.bash_output "/etc/init.d/portmap status" $["TERM":"raw"] $["stderr":"", "stdout":"No Process"]
Execute	.target.bash_output "/etc/init.d/ypbind status" $["TERM":"raw"] $["stderr":"", "stdout":"No Process"]
Execute	.target.bash_output "/etc/init.d/portmap status" $["TERM":"raw"] $["stderr":"", "stdout":"No Process"]
Execute	.target.bash_output "/etc/init.d/ypbind status" $["TERM":"raw"] $["stderr":"", "stdout":"No Process"]
Return	nil
